models/QuizModel.php

```diff
@@ -461,7 +461,7 @@ public static function get_quiz_attempts( $start = 0, $limit = 10, $search_filte
 					INNER JOIN {$wpdb->posts} quiz ON quiz_attempts.quiz_id = quiz.ID
 					INNER JOIN {$wpdb->users} AS users ON quiz_attempts.user_id = users.ID
 					INNER JOIN {$wpdb->posts} AS course ON course.ID = quiz_attempts.course_id
-					INNER JOIN {$wpdb->prefix}tutor_quiz_attempt_answers AS ans ON quiz_attempts.attempt_id = ans.quiz_attempt_id
+					-- INNER JOIN {$wpdb->prefix}tutor_quiz_attempt_answers AS ans ON quiz_attempts.attempt_id = ans.quiz_attempt_id
```

Comment:
Why -- added?

Reply:
SQL `--comment out` Removed the INNER JOIN with `tutor_quiz_attempt_answers` to include timed-out attempts as well.